### PR TITLE
Create Voters Guide 2017

### PR DESCRIPTION
**Pull request by parispittman**
@castrojo

### CODE REVIEW

File: voters-guide-2017.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Doug Davis |	IBM	| [@duglin](https://github.com/duglin)
+Ihor Dvoretskyi |	Mirantis |	[@idvoretskyi](https://github.com/idvoretskyi)
+Ilya Dmitrichenko |	Weave |	[@errordeveloper](https://github.com/errordeveloper)
+Jaice Singer DuMars |	Microsoft	| [@jdumars](https://github.com/jdumars)
+Justin Santa Barbara	|	| [@justinsb](https://github.com/justinsb)
+Kris Nova |	Microsoft	| [@kris-nova](https://github.com/kris-nova)
+Matt Farina |	Samsung SDS	 | [@mattfarina](https://github.com/mattfarina)
+Michael Rubin | Google |	[@mrubin](https://github.com/mrubin)
+Michelle Noorali |	Microsoft |	[@michelleN](https://github.com/michelleN)
+Phillip Whitrock |	Google | 	[@pwittrock](https://github.com/pwittrock)
```
Comment (zhangxiaoyu-zidif) [inline voters-guide-2017.md]: I think his name is “Phillip Wittrock”
/cc @pwittrock

File: voters-guide-2017.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+Aaron Schlesinger |	Microsoft |	[@arschles](https://github.com/arschles)
+Adnan Abdulhussein |	Bitnami |	[@prydonius](https://github.com/prydonius)
+Alex Pollitt |	Tigera	| [@lxpollitt](https://github.com/lxpollitt)
+Caleb Miles | CoreOS |	[@calebamiles](https://github.com/calebamiles)
+Derek Carr |	Red Hat |	[@derekwaynecarr](https://github.com/derekwaynecarr)
+Doug Davis |	IBM	| [@duglin](https://github.com/duglin)
+Ihor Dvoretskyi |	Mirantis |	[@idvoretskyi](https://github.com/idvoretskyi)
+Ilya Dmitrichenko |	Weave |	[@errordeveloper](https://github.com/errordeveloper)
+Jaice Singer DuMars |	Microsoft	| [@jdumars](https://github.com/jdumars)
+Justin Santa Barbara	|	| [@justinsb](https://github.com/justinsb)
```
Comment (xiangpengzhao) [inline voters-guide-2017.md]: Justin is from FathomDB. ping @justinsb for confirmation :)

File: voters-guide-2017.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+## DECISION
+The newly elected body will be announced in the weekly Kubernetes Community Meeting on October 5, 2017 at 10:00am US Pacific Time. [Please join us](https://groups.google.com/forum/#!forum/kubernetes-community-video-chat).
+
+Immediately following the meeting, the raw voting results and winners will be published on the [Kubernetes Blog](http://blog.kubernetes.io/).
+
+For more information, definitions, and/or detailed election process, see full [steering committee charter](https://github.com/kubernetes/steering/blob/master/charter.md).
+
+# NOMINEES
+Don’t know someone on this list? Check out their K8s bio by first+last name [here](https://github.com/kubernetes/community/tree/master/community/elections/2017). 
+
```
Comment (xiangpengzhao) [inline voters-guide-2017.md]: Maybe we can add a column to link to every nominees' bio link?

Comment (parispittman) [reply voters-guide-2017.md]: will add the link to the bios in the name section as they come in

File: voters-guide-2017.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+Name | Organization/Company | GitHub 
+--- | --- | --- 
+Aaron Crickenberger	| Samsung SDS |	[@spiffxp](https://github.com/spiffxp)
+Aaron Schlesinger |	Microsoft |	[@arschles](https://github.com/arschles)
+Adnan Abdulhussein |	Bitnami |	[@prydonius](https://github.com/prydonius)
+Alex Pollitt |	Tigera	| [@lxpollitt](https://github.com/lxpollitt)
+Caleb Miles | CoreOS |	[@calebamiles](https://github.com/calebamiles)
+Derek Carr |	Red Hat |	[@derekwaynecarr](https://github.com/derekwaynecarr)
+Doug Davis |	IBM	| [@duglin](https://github.com/duglin)
+Ihor Dvoretskyi |	Mirantis |	[@idvoretskyi](https://github.com/idvoretskyi)
```
Comment (idvoretskyi) [inline voters-guide-2017.md]: I don't work for Mirantis anymore. Please, remove this affiliation from the list.

**Pull request merged**